### ISSUE TITLE
[P2][Sourcing][Accessibility]Table: Frozen column status is not being conveyed by screenreader

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11202]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
In UI5 , frozen columns are read as "Fixed" but the same is not the case with NGX table columns.
No information is mentioned whether the current column is frozen or not by the screenreader for disabled users

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/ platform: 0.43.24

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
1) use the example https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#freezable
2) Traverse through frozen columns and notice the screenreader

In UI5 example https://sapui5.hana.ondemand.com/#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.TableFreeze the "Fixed" word is read when focus is on frozen column , which is helpful for disabled users 

<img width="1728" alt="Screenshot 2024-01-09 at 1 54 54 PM" src="https://github.com/SAP/fundamental-ngx/assets/88182358/19f18bcd-b0f0-4b11-b5f1-c986ba07b256">

#### Please provide relevant source code (if applicable).
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#freezable
#### Please provide stackblitz example(s).
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#freezable
#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Discussed with ATL team and reported

**Comment by sagar7259** [comment]
Hello,

The state of the column weather they are frozen or unfrozen should be conveyed to the users or else users will not understand that the columns are already frozen.

Regards,
Sagar

**Comment by droshev** [comment]
@nikolay-kolarov do you think you can advise us here? thanks

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

The short answer is to add the semantic.
Unfortunately, you do not have an aria-labelledby calculation on focus level in a similar way like in UI5 (and there the fixed info is putted there). And there is no html/aria attribute that can serve the means to indicate a fixed column.
So if you do not want to have an aria-labelledby calculation, the alternative that you might wanna try is to put it in aria-describedby. And you have to check if this is reliably announced by the screen readers. In this way you will keep the accessibleName calculation that you have at the moment and add the fixed info through the description.

@simlin - Do you have by chance any other suggestion and opinion here? 

Best Regards,
Nikolay
